### PR TITLE
[CARBONDATA-346] Add unit test for CarbonUtil

### PR DESCRIPTION
**Pull request by kunal642**
Added unit test for CarbonUtil

### CODE REVIEW

**Comment by jackylk** [comment]
Please correct the PR title as other PR and provide description of the PR

**Comment by Zhangshunyu** [comment]
Please add APACHE LICENSE for new files.

**Comment by kunal642** [comment]
@jackylk @Zhangshunyu: Implemented the changes you asked for.
Please review
Added unit test for ValueCompressionUtil as well

**Comment by jackylk** [comment]
Please modify the description of this PR, you should describe what test case are added. And it is better to separate them into different PR each one below 500 lines, because this PR is too big

**Comment by kunal642** [comment]
@jackylk Removed ValueCompressionUtilTest from this PR.
CarbonUtil had a lot of test cases so i cannot make it below 500 lines.

**Comment by ravipesala** [comment]
Please don't write testcases for non business logic classes like exception classes.
And also format the new code using formatter available in dev package

File: core/src/test/java/org/apache/carbondata/core/util/CarbonUtilTest.java [hunk 1]
```diff
@@ -18,18 +18,746 @@
...
+    public void testCloseStreams() throws IOException {
+        FileReader stream = new FileReader("../core/src/test/resources/testFile.txt");
+        BufferedReader br = new BufferedReader(stream);
+        CarbonUtil.closeStreams(br);
+        br.ready();
+    }
+
+    @Test
+    public void testToGetCardinality() {
+        int result = CarbonUtil.getIncrementedCardinality(10);
```
Comment (ravipesala) [inline core/src/test/java/org/apache/carbondata/core/util/CarbonUtilTest.java]: add more checks by passing more different values here.

**Comment by kunal642** [comment]
@ravipesala Please review the changes

**Comment by ravipesala** [comment]
LGTM

**Pull request closed without merging**